bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -115,11 +115,15 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <use_c14>.false.</use_c14>
 <use_c13_timeseries>.false.</use_c13_timeseries>
 <use_c14_bombspike>.false.</use_c14_bombspike>
+<!-- Turn timeseries on if Carbon isotopes are on -->
+<use_c13_timeseries use_c13=".true.">.true.</use_c13_timeseries>
+<use_c14_bombspike use_c14=".true.">.true.</use_c14_bombspike>
 
-<use_c13 phys="clm6_0" bgc_mode="bgc" >.true.</use_c13>
-<use_c14 phys="clm6_0" bgc_mode="bgc" >.true.</use_c14>
-<use_c13_timeseries phys="clm6_0" bgc_mode="bgc" >.true.</use_c13_timeseries>
-<use_c14_bombspike phys="clm6_0" bgc_mode="bgc" >.true.</use_c14_bombspike>
+<!-- Turn Carbon isotopes on for clm6_0 physics with BGC on with CRUJRA or CAM7 forcing and for historical period-->
```

Comment:
@wwieder and @slevis-lmwg how does this sound as the restriction to use for use_c13/c14 to restrict when it's turned on more than before?